### PR TITLE
Remove deprecated geoip feature

### PR DESCRIPTION
**Pull request by ruflin**
Geoip was deprecated in 5.0 and is now removed for 6.0.

### CODE REVIEW

**Comment by ruflin** [comment]
@monicasarbu Would be great to get your input on this one. Are there other parts that should be removed related to topology?

**Comment by tsg** [comment]
Needs a rebase now.

**Comment by ruflin** [comment]
@tsg rebased

**Comment by tsg** [comment]
Is this one still in progress?

**Comment by ruflin** [comment]
@tsg nope, forgot to change the label.

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -31,6 +31,7 @@ https://github.com/elastic/beats/compare/v5.1.1...master[Check the HEAD diff]
 - Change fieldnames couchbase.node.couch.*.actual_disk_size.* to couchbase.node.couch.*.disk_size.* {pull}3545[3545]
 
 *Packetbeat*
+- Remove deprecated geoip
```
Comment (tsg) [inline CHANGELOG.asciidoc]: Can you add the PR number, please?

**Comment by tsg** [comment]
There's a set of geoip files in the system tests which can be probably deleted as well:

```
./packetbeat/tests/system/files/geoip_city.dat
./packetbeat/tests/system/files/geoip_city_blocks.csv
./packetbeat/tests/system/files/geoip_city_loc.csv
```

File: CHANGELOG.asciidoc (hunk 1 above)
Comment (ruflin) [inline CHANGELOG.asciidoc]: done

**Pull request merged**